components/scream/src/physics/shoc/shoc_energy_fixer_disp.cpp

```diff
@@ -0,0 +1,81 @@
+#include "shoc_functions.hpp"
+
+#include "ekat/kokkos/ekat_subview_utils.hpp"
+
+namespace scream {
+namespace shoc {
+
+template<>
+void Functions<Real,DefaultDevice>
+::shoc_energy_fixer_disp(
```

Comment:
My personal taste here would be for spelling out `dispatch`, but that's very subjective.

Reply:
Those extra 4 keystrokes would give me carpal tunnel!